awx/ui/client/lib/components/popover/popover.partial.html

```diff
@@ -1,16 +1,16 @@
 <div ng-show="popover.text" class="at-Popover" ng-class="{ 'at-Popover--inline': popover.inline }">
-    <span class="at-Popover-icon"
+    <div class="at-Popover-icon"
           ng-class="{ 'at-Popover-icon--defaultCursor': popover.on === 'mouseenter' && !popover.click }">
         <i class="fa {{ popover.icon }}"></i>
-    </span>
+    </div>
     <div class="at-Popover-container">
         <div class="at-Popover-arrow">
-            <i ng-if="popover.position === 'right'" class="fa fa-caret-left fa-2x"></i>
-            <i ng-if="popover.position === 'top'" class="fa fa-caret-down fa-2x"></i>
+            <div ng-if="popover.position === 'right'" class="fa fa-caret-left fa-2x"></div>
+            <div ng-if="popover.position === 'top'" class="fa fa-caret-down fa-2x"></div>
         </div>
         <div class="at-Popover-content">
-            <h4 ng-if="popover.title" class="at-Popover-title">{{ popover.title | translate }}</h4>
-            <p ng-if="popover.text" class="at-Popover-text">{{ popover.text | translate}}</p>
+            <h4 ng-if="popover.title" class="at-Popover-title">{{ popover.title }}</h4>
+            <p ng-if="popover.text" class="at-Popover-text">{{ popover.text }}</p>
```

Comment:
As of right now, I don't think we should remove these translations, b/c it will result in popover text from being translated. Related issue: https://github.com/ansible/ansible-tower/issues/5361